views/pages/toplevel/home.njk

```diff
@@ -25,75 +28,73 @@
 {% block content %}
 
     {% set heroContent %}
-        {{ headers.overlayText('h2', "The Big Lottery Fund is the largest funder of community activities in the UK. We distribute money raised by players of the <a href=\"https://www.national-lottery.co.uk\">National Lottery</a>", 'overlay-text--has-links t2') }}
+        <div class="o-button-group">
+            <a class="btn btn--small [ accent--white a--btn ]"
+                href="{{ buildUrl('toplevel', 'under10k') }}">
+                <span class="accent--pink a--text">
+                    <span class="u-wide-only">Learn how to apply for funding under £10,000</span>
+                    <span class="u-not-wide">Apply for funding under £10,000</span>
+                </span>
+            </a>
+
+            <a class="btn btn--small [ accent--white a--btn ]"
+                href="{{ buildUrl('toplevel', 'over10k') }}">
+                <span class="accent--blue a--text">
+                    <span class="u-wide-only">Learn how to apply for funding over £10,000</span>
+                    <span class="u-not-wide">Apply for funding over £10,000</span>
+                </span>
+            </a>
+        </div>
     {% endset %}
 
-    {{ hero.header('home', heroContent, 'blue', 'The Sea Cadet Corps', extraClasses = 'spaced') }}
+    {{
+        hero.superHero(
+            name = 'home-super',
+            titleDefault = 'The Big Lottery Fund distributes <a href="#">over £600m</a> a year to communities
+                across the UK, raised by players of the <a href="https://www.national-lottery.co.uk/">National Lottery</a>.',
+            titleShort = 'The Big Lottery Fund distributes money raised by players of the
+                <a href="https://www.national-lottery.co.uk/">National Lottery</a> to communites across the UK.',
+            content = heroContent,
+            caption = 'Cloughmills Community Action, Grant £4,975*'
+        )
+    }}
 
     <article role="main">
-        <div class="inner">
```

Comment:
I'm scope creeping a little bit here my moving other parts of the homepage around to match the designs as part of this PR happy to pull these out into separate PRs.